packages/flame/lib/src/game/game.dart

```diff
@@ -20,7 +21,7 @@ import 'projector.dart';
 ///
 /// Subclass this to implement the [update] and [render] methods.
 /// Flame will deal with calling these methods properly when the game's widget is rendered.
-abstract class Game extends Projector {
+abstract class Game extends Component implements Projector {
```

Comment:
Since `Game` is now a component, it should probably declare itself to be `_isMounted`